test/CppTest_UWP/Package.appxmanifest

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Package xmlns="http://schemas.microsoft.com/appx/manifest/foundation/windows10" xmlns:mp="http://schemas.microsoft.com/appx/2014/phone/manifest" xmlns:uap="http://schemas.microsoft.com/appx/manifest/uap/windows10" IgnorableNamespaces="uap mp">
+  <Identity
+    Name="1ea68fdd-88ac-47b0-ba8a-747427ad986c"
+    Publisher="CN=Jon"
```

Comment:
Fix these to have the a Microsoft publisher string

Reply:
I notice you didn't check in the PFX -- how can someone else build and sign this?

Reply:
The package has this marker:

```xml
    <AppxPackageSigningEnabled>false</AppxPackageSigningEnabled>
```

And the resulting package is not signed when registered to run from layout.  This test app isn't useful beyond testing, but "Deploy" does actually put it in your start menu.